mage_ai/data_preparation/models/block/dbt/utils/__init__.py

```diff
@@ -175,6 +188,8 @@ def update_model_settings(block, upstream_blocks, upstream_blocks_previous):
             )
 
             settings = load_sources(block)
+
+            print('WTFFFFFFFFFFFFFF', new_source)
```

Comment:
@tommydangerous could you remove this?

Reply:
LOL!

Reply:
https://github.com/mage-ai/mage-ai/pull/1812